runners/google-cloud-dataflow-java/worker/src/test/java/org/apache/beam/runners/dataflow/worker/StreamingDataflowWorkerTest.java

```diff
@@ -797,9 +806,30 @@ private StreamingDataflowWorker makeWorker(
       Supplier<Instant> clock,
       Function<String, ScheduledExecutorService> executorSupplier,
       int localRetryTimeoutMs) {
+    return makeWorker(
+        ImmutableMap.of(),
+        instructions,
+        options,
+        publishCounters,
+        clock,
+        executorSupplier,
+        localRetryTimeoutMs);
+  }
+
+  private StreamingDataflowWorker makeWorker(
```

Comment:
there are 6 makeWorkers, it seems some autovalue builder in this test would make it more readable and less duplication of the default for various params.

Reply:
done used a builder with some test parameters